item-api/src/main/java/com/example/itemapi/interfaces/presentation/ItemController.java

```diff
@@ -29,4 +29,9 @@ public ItemInfoResponse getBy(@PathVariable(name = "itemId") Long itemId) {
     public void decreaseStock(@PathVariable Long itemId, @RequestBody ItemRequest.DecreaseStock decreaseStock) {
         itemService.decreaseStock(itemId, decreaseStock.decreaseCount());
     }
+
+    @DeleteMapping("/items/{itemId}")
+    public void deleteBy(@PathVariable Long itemId) {
+        itemService.deleteBy(itemId);
```

Comment:
```suggestion
        itemService.deleteBy(itemId);
        return ResponseEntity.ok();
```
기본적인 응답 규격은 표준에 맞춰서 응답해주면 좋을것 같아요~!

Reply:
넵! 수정하겠습니다.